packages/core/src/shared/logger/logger.ts

```diff
@@ -3,8 +3,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+// Disable because this is the main logger file.
+/* eslint-disable aws-toolkits/no-console-log */
```

Comment:
can we put this in the `ConsoleLogger` scope? shouldn't be needed for the rest of this file.